mapslang/scr_actn.c

```diff
@@ -38,13 +38,19 @@
 #include "textmenu.h"
 #include "scr_txtgen.h"
 
+extern void actions_mdtileset(struct SCRMODE_DATA *scrmode,struct WORKMODE_DATA *workdata,int key);
+extern void draw_mdtileset(struct SCRMODE_DATA *scrmode,struct WORKMODE_DATA *workdata);
+extern void start_mdtileset(struct SCRMODE_DATA *,struct WORKMODE_DATA *);
+extern void end_mdtileset(struct SCRMODE_DATA *,struct WORKMODE_DATA *);
+extern short color_mdtileset(struct SCRMODE_DATA *scrmode, struct MAPMODE_DATA *mapmode, struct LEVEL *lvl, int tx, int ty);
```

Comment:
Create a `scr_tileset.h` header file and declare these function there instead of using `extern`

Reply:
ok

Reply:
done